catalog/spatial/geocoding/spatial-geocoding-create/src/test/java/org/codice/ddf/spatial/geocoding/create/TestGeoNamesCreator.java

```diff
@@ -33,7 +33,8 @@ public void testNoEmptyFields() {
                 -111.84125,
                 "PPL",
                 236123,
-                "Candler,Candleris,Chandler,Chandlur");
+                "Candler,Candleris,Chandler,Chandlur",
+                "US");
```

Comment:
That same spec includes alpha 3 country codes which is what we want to use in DDF.  See https://codice.atlassian.net/wiki/display/DDF/location.country-code